packages/frontend-2/components/project/page/settings/general/block/Access.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <ProjectPageSettingsBlock
+    background
+    title="Access"
+    :icon="LockClosedIcon"
+    :disabled-message="
+      disabled && disabledMessage ? 'You must be a project owner' : undefined
```

Comment:
The message will never be rendered unless you pass in the 'disabledMessage' prop, is that expected?